examples/eventwebhook/eventwebhook.go

```diff
@@ -35,9 +35,9 @@ func EnableSecureWebhook() {
 	}
 }
 
-// GetPublicKeyForWebhook : Get Public Key for Event Webhook.
+// GetPublicKeyForSecureWebhook : Get Public Key for Event Webhook.
```

Comment:
could we call this `GetPublicKeyForSignedWebhookRequests` to make it consistent with product and how it's presented in the UI?